site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -9,19 +9,22 @@ has_warning: One or more of these permissions <a href="/docs/extensions/mv3/perm
 
 ## Manifest {: #manifest }
 
-Extensions must declare either the `declarativeNetRequest` or the
-`declarativeNetRequestWithHostAccess` (available since **Chrome 96**) permission in the extension
-[manifest][1] to use this API. The former allows extensions to block and upgrade requests without
-any [host permissions][2]. Host permissions are still required if the extension wants to redirect a
-request or modify headers on it. The `declarativeNetRequestWithHostAccess` permission always
-requires host permissions to the request URL and initiator to act on a request.
+### Permissions {: #permission-definitions }
 
-The `declarativeNetRequestFeedback` permission is required to access functions and events which
-return information on declarative rules matched.
+`declarativeNetRequest`
+: Allows extensions to block and upgrade requests.
```

Comment:
With how this is currently written, it implies to me that `declarativeNetRequest` is less capable than `declarativeNetRequestWithHostAccess`. Could we clarify that with sufficient host permissions both are equivalent (and can redirect requests and modify headers)?

Reply:
We can, but I need some clarification because this definition regurgitates the current definition. Your explanation makes it sound as though there's no difference between the permissions. 

What's the difference between the two permissions?

Reply:
Hopefully our discussion yesterday was helpful :)

Reply:
I took a look at the updated summary and I think it would still be nice to work on. It says "both require host permissions" which is not true - "declarativeNetRequestWithHostAccess" doesn't require anything, but for it to be useful you have to separately request host permissions for the sites you're interested in.